components/Account/Memberships/Claim.js

```diff
@@ -238,6 +238,11 @@ class ClaimMembership extends Component {
       'memberships/claim/body'
     ], {}, false)
 
+    const contextAddendum = t.first([
+      `memberships/claim/${context}/addendum`,
+      'memberships/claim/addendum'
+    ], {}, false)
```

Comment:
`missingValue` should be a string. Without or empty replacements it works with other values too but if one adds replacements later it will explode.

```suggestion
    ], undefined, '')
```